setup.py

```diff
@@ -173,7 +173,11 @@ def do_setup():
               'theano.misc': ['*.sh'],
               'theano.d3viz' : ['html/*','css/*','js/*']
           },
-          scripts=['bin/theano-cache', 'bin/theano-nose', 'bin/theano-test'],
+          entry_points={
+              'console_scripts': ['theano-cache = bin.theano_cache:main',
+                                  'theano-nose = bin.theano_nose:main',
+                                  'theano-test = bin.theano_test:main']
```

Comment:
@lamblin Yes, it works (on my computer, at least)!

It's a bit weird, but when I put the scripts into theano folder, theano is fully imported (from `theano/__init__.py`) before the code of the scripts is loaded, then THEANO_FLAGS is parsed during this importation and no more into the script code. However I need to modify THEANO_FLAGS before it is parsed, to add `cxx=` to it, specifically for `theano-cache` script. I have not be able to do it into the theano folder.

THEANO_FLAGS is modified at the top of the `theano-cache` script before the `import theano` line.